src/BrowserView.m

```diff
@@ -139,8 +153,19 @@ -(void)createNewTabWithView:(NSView<BaseView> *)newTabView makeKey:(BOOL)keyIt
 {
 	NSTabViewItem *tabViewItem = [[NSTabViewItem alloc] initWithIdentifier:newTabView];
 	tabViewItem.view = newTabView;
+
 	[self.tabView addTabViewItem:tabViewItem];
 
+	//newly created item will be selected first or last to be selected
+	if (self.selectNewItemFirst)
```

Comment:
Shouldn't it be `if (self.selectPreviousOnClose && keyIt && self.selectNewItemFirst)` ?

Reply:
This is implicit, because nothing will happen when the tabOrdering array is nil.

Reply:
I think however that tabs opened in background should be treated differently from tabs directly selected on open.

Reply:
I see what you mean. Actually I overlooked that you wrote `keyIt`. If you prefer, for readability, that `keyIt` is a condition here I can add it, otherwise this is taken care of by the callback from the tabView when calling `showTabItemView`: The tab will be moved to the last position in the array, which means it was opened last.